diff --git a/db/migrate/20150922083705_create_users.rb b/db/migrate/20150922083705_create_users.rb
index 226ea7a..2b1ccb8 100644
--- a/db/migrate/20150922083705_create_users.rb
+++ b/db/migrate/20150922083705_create_users.rb
@@ -1,6 +1,6 @@
 class CreateUsers < ActiveRecord::Migration[4.2]
   def change
-    create_table :users do |t|
+    create_table :users, options: 'ROW_FORMAT=DYNAMIC' do |t|
       t.string :name
 
       t.timestamps null: false
diff --git a/db/migrate/20150922083929_create_calendars.rb b/db/migrate/20150922083929_create_calendars.rb
index 57f1548..355ef1b 100644
--- a/db/migrate/20150922083929_create_calendars.rb
+++ b/db/migrate/20150922083929_create_calendars.rb
@@ -1,6 +1,6 @@
 class CreateCalendars < ActiveRecord::Migration[4.2]
   def change
-    create_table :calendars do |t|
+    create_table :calendars, options: 'ROW_FORMAT=DYNAMIC' do |t|
       t.string :name
       t.text :propxml
       t.references :user, index: true, foreign_key: true
diff --git a/db/migrate/20150922112809_create_schedules.rb b/db/migrate/20150922112809_create_schedules.rb
index d226eee..88894fc 100644
--- a/db/migrate/20150922112809_create_schedules.rb
+++ b/db/migrate/20150922112809_create_schedules.rb
@@ -1,6 +1,6 @@
 class CreateSchedules < ActiveRecord::Migration[4.2]
   def change
-    create_table :schedules do |t|
+    create_table :schedules, options: 'ROW_FORMAT=DYNAMIC' do |t|
       t.string :type
       t.text :summary
       t.datetime :date_start
diff --git a/db/migrate/20151209235553_create_changes.rb b/db/migrate/20151209235553_create_changes.rb
index c4e430f..06cedb3 100644
--- a/db/migrate/20151209235553_create_changes.rb
+++ b/db/migrate/20151209235553_create_changes.rb
@@ -1,6 +1,6 @@
 class CreateChanges < ActiveRecord::Migration[4.2]
   def change
-    create_table :changes do |t|
+    create_table :changes, options: 'ROW_FORMAT=DYNAMIC' do |t|
       t.references :calendar, index: true, foreign_key: true
       t.string :uri
       t.boolean :is_delete
