--- a/hw/i386/multiboot.c	2018-05-25 16:28:49.000000000 +0900
+++ b/hw/i386/multiboot.c	2018-05-25 16:29:07.000000000 +0900
@@ -193,10 +193,12 @@
         int kernel_size;
         fclose(f);

+#if 0
         if (((struct elf64_hdr*)header)->e_machine == EM_X86_64) {
             error_report("Cannot load x86-64 image, give a 32bit one.");
             exit(1);
         }
+#endif

         kernel_size = load_elf(kernel_filename, NULL, NULL, &elf_entry,
                                &elf_low, &elf_high, 0, I386_ELF_MACHINE,
